### PR TITLE
"Improved" VLAN filters documentation

### PR DESCRIPTION
**Pull request by dwlfrth**
# Description

More structured VLAN filters configuration documentation
# Impacts

None
# Delete branch after merge

YES

### CODE REVIEW

File: conf/vlan_filters.conf.example [hunk 1]
```diff
@@ -1,10 +1,191 @@
...
+#
+#
+# A "VLAN filter" configuration is mainly constituted of two (2) parts; rules(s)/condition(s) and action(s)
+# Some working examples covering useful use cases can be found at the bottom of this file
+#
+#
+# Rule(s)/Condition(s)
+# ------------------
+# Usually, a rule/condition contains a filter, an operator, in some cases an attributes, and a value
+# Some working examples covering useful use cases can be found at the bottom of this file
```
Comment (fdurand) [inline conf/vlan_filters.conf.example]: You repeat twice this line

File: conf/vlan_filters.conf.example [hunk 2]
```diff
@@ -1,10 +1,191 @@
...
+# Operator can be:
+#     is
+#     is_not
+#     match
+#     match_not
+#     defined
+#     not_defined
+#     regex
+#     date_is_before
+#     date_is_after
```
Comment (fdurand) [inline conf/vlan_filters.conf.example:124]: Miss time

File: conf/vlan_filters.conf.example [hunk 3]
```diff
@@ -1,10 +1,191 @@
...
+# Note: Time / Date based rule(s)/condition(s) values follows the following logic:
+# http://search.cpan.org/~pryan/Period-1.20/Period.pm
+# For example, if you want to allow access from Monday to Friday, 9am to 11am and from 2pm to 4pm:
+# wd {Mon Tue Wed Thu Fri} hr {9am-11am}, wd {Mon Tue Wed Thu Fri} hr {2pm-4am}
+#
+#
+# Actions
+# -------
+# Usually, an action contains a scope, a role and optionally, a action with action parameters
+# Some working examples covering useful use cases can be found at the bottom of this file
```
Comment (fdurand) [inline conf/vlan_filters.conf.example]: You repeat again this line

File: conf/vlan_filters.conf.example [hunk 4]
```diff
@@ -65,10 +240,10 @@
 # scope = AutoRegister
 # role = default
 #
-# This will autoregister machine auth on ethernet EAP or EAP-TLS connection and refuse 
+#
+# - This will autoregister machine auth on ethernet EAP or EAP-TLS connection and refuse
```
Comment (fdurand) [inline conf/vlan_filters.conf.example:239]: You have to change refuse role to REJECT

**Comment by dwlfrth** [comment]
Adjustments made

**Pull request merged**